server/src/main/scala/org/http4s/server/middleware/URITranslation.scala

```diff
@@ -3,17 +3,18 @@ package server
 package middleware
 
 import cats.Functor
+import cats.data.Kleisli
 
 object URITranslation {
-  def translateRoot[F[_]: Functor](prefix: String)(
-      @deprecatedName('service) routes: HttpRoutes[F]): HttpRoutes[F] = {
+  def translateRoot[F[_], G[_]: Functor, A](prefix: String)(
```

Comment:
I think I called this `B` in another one, just so response was always `B`.  But I also thought about just using the first available letter like this.  They're obviously equivalent.

Does anyone have a strong preference?  My primary preference is only for consistency.  I guess maybe I like the style here better.

Reply:
No preference here. With you on the consistency front, so am happy to change!